scripts/Muon/GUI/Common/fitting_tab_widget/fitting_tab_presenter.py

```diff
@@ -53,4 +145,25 @@ def selected_data(self):
     @selected_data.setter
     def selected_data(self, value):
```

Comment:
can we have a better name than value. From the name I assume it is the name of a single ws, but looking at the code it seems to be a list